client/checkout/classic/upe.js

```diff
@@ -287,6 +287,7 @@ jQuery( function ( $ ) {
 						paymentMethodsConfig[ event.value.type ].isReusable;
 					showNewPaymentMethodCheckbox( isPaymentMethodReusable );
 					isUPEComplete = event.complete;
+					console.log( isUPEComplete );
```

Comment:
This might have snuck in from a develop merge.

Reply:
Nice catch! If I've forgotten a full-stop at the end of a doc comment this linter won't let me hear the end of it, but if I'm actually about to stroll out of the house with my trousers down this linter is happy to let the whole world watch. Just deleted this line and cleared some of the egg from my face.